src/components/Api.js

```diff
@@ -0,0 +1,179 @@
+export class Api {
+  constructor(config) {
+    this.url = config.baseUrl;
+    this.headers = config.headers;
+  }
+
+  getInitialCards() {
+    return fetch(`${this.url}/cards`, {
+      headers: {
+        authorization: "fe7e07a4-81c5-490b-807b-e6a7cec619a0",
+      },
+    }).then((res) => {
+      if (res.ok) {
+        return res.json();
+      }
+      // if the server returns an error, reject the promise
+      return Promise.reject(`Error: ${res.status}`);
+    });
+  }
+
+  getUserInfo() {
+    return fetch("https://around-api.en.tripleten-services.com/v1/users/me", {
+      method: "GET",
+      headers: {
```

Comment:
Needs correction: you should use the saved headers object (this._headers) in every api method

Reply:
I updated the api methods to include "this._headers.